AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRiSkyboxAdded.py

```diff
@@ -136,36 +145,57 @@ def AtomEditorComponents_HDRiSkybox_AddedToEntity():
             AtomComponentProperties.hdri_skybox('Cubemap Texture'))
         Report.result(Tests.cubemap_property_set, get_cubemap_property == skybox_cubemap_material_asset.id)
 
+        properties = hdri_skybox_component.get_property_type_visibility()
+        for path, value in properties.items():
+            Report.info(f"'{path}' {value}")
```

Comment:
this is for debugging to get the component property paths. you probably don't want to ship this

Reply:
fixed